### PR TITLE
Report AGS with new time/timeSpecificity API

### PR DESCRIPTION
**Pull request by brmscheiner**

### CODE REVIEW

File: src/constants/timeSpecificityChoices.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const timeSpecifityChoices = [
+  {
+    label: 'I know the year',
```
Comment (brmscheiner) [inline src/constants/timeSpecificityChoices.js:3]: not worried about translations on these rn just trying to get a fix in ASAP

File: src/pages/reportSighting/ReportForm.jsx [hunk 2]
```diff
@@ -336,9 +301,7 @@ export default function ReportForm({
               const formValid =
                 nextIncompleteFields.length === 0 &&
                 acceptedTerms &&
-                durationAcceptable &&
-                oneLocationFieldPresent &&
-                !startTimeAfterEndTime;
+                oneLocationFieldPresent;
```
Comment (Atticus29) [inline src/pages/reportSighting/ReportForm.jsx:304]: We still want time to be required for an AGS, right?

**Review by Atticus29 — COMMENTED** [review]
Other than the possible issue of requiring Time in the sighting report form, LGTM.

Are you sure formatHoustonTime works?
formatISO was failing for me when I committed an AGS with the "....Z" error.
But if this works, cool!
And that's QA's job.

File: src/pages/reportSighting/utils/prepareReport.js [hunk 3]
```diff
@@ -27,6 +28,7 @@ export function prepareBasicReport(
       : assetReferences,
     decimalLatitude: sightingData.gps[0],
     decimalLongitude: sightingData.gps[1],
+    time: formatHoustonTime(get(sightingData, 'time')),
```
Comment (Atticus29) [inline src/pages/reportSighting/utils/prepareReport.js:31]: Looks like timeSpecificity is missing from this part??

**Review by Atticus29 — COMMENTED** [review]
One potential deal-breaking thing, but otherwise LGTM.

**Pull request merged**